client/packages/requisitions/src/RequestRequisition/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -81,7 +87,23 @@ const useStatusChangeButton = () => {
   const onConfirmStatusChange = async () => {
     if (!selectedOption) return null;
     try {
-      await update({ status: selectedOption.value });
+      let updatedComment = comment;
```

Comment:
looks good although could be a method to avoid `let` (I don't mind let if variable is just changed once and it's clearly visible, only commenting since it's been pointed out in my on PR's that it's best not to mutate variable):
```
await update({ status: selectedOption.value, comment: getUpdatedComment(comment) });
```